pom.xml

```diff
@@ -69,7 +69,7 @@
   </developers>
 
   <properties>
-    <vertx.version>3.9.4</vertx.version>
+    <vertx.version>4.0.0</vertx.version>
```

Comment:
The vertx version update breaks much of our HttpClient code in the handler package (see the Travis build output).  Can we roll back this update and accept the other changes?

Reply:
I'd say we ignore the whole thing for now. Snyk opened this -- I guess I need to update its connection to use a UCLA bot account. I find it interesting that Snyk is doing reformatting now though... I don't think I've seen that before.